README.md

```diff
@@ -176,7 +176,7 @@ This example is based on a real life [OSS agent we've shipped to manage our depl
 ![035-deploybot-conversation](./img/035-deploybot-conversation.png)
 
 
-We haven't given this agent a huge pile of tools or tasks. The primary value in the LLM is parsing the human's plaintext feedback and proposing an updated course of action. We isolate tasks and contexts as much as possible to keep
+We haven't given this agent a huge pile of tools or tasks. The primary value in the LLM is parsing the human's plaintext feedback and proposing an updated course of action. We isolate tasks and contexts as much as possible to keep SOMETHING IS MISSING HERE.
```

Comment:
Everything else in this PR is minor nits mostly, but this one is probably worth addressing.